includes/widgets/image-box.php

```diff
@@ -155,12 +155,13 @@ protected function register_controls() {
 			]
 		);
 
-		$this->add_control(
+		$this->add_responsive_control(
 			'position',
 			[
 				'label' => esc_html__( 'Image Position', 'elementor' ),
 				'type' => Controls_Manager::CHOOSE,
 				'default' => 'top',
+				'mobile_default' => 'top',
 				'options' => [
```

Comment:
Default mobile version for BC reasons.

Reply:
@hein-obox Maybe it is worth adding a `tablet_default`? I see most places in Elementor has default/tablet_default/mobile_default?

Reply:
I am hesisant to do this for BC reasons. 

@avivu Please advise.

Reply:
@hein-obox @louiswol94 
If there is a default today, we should keep it, but if there is not a default today, we should leave it as it is.

Reply:
@hein-obox @avivu The control was not responsive before so `tablet_default` would not have been there.
What I meant was that Hein JUST added `mobile_default`, and nothing for `tablet_default` after making the control responsive, and I see in Elementor we usually specify BOTH. But I think it is okay, Udi will pick it up if it is a problem.
Thanks!

Reply:
@hein-obox Did you see this question?